packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -29,34 +34,44 @@ export function FormWrapper({
   leftButtonProps = {},
   rightButtonProps = {},
   title,
+  isTitleBesideCancel,
   headerRef,
 }: FormWrapperProps): ReactElement {
-  const containerClassName =
-    typeof className === 'string' ? className : className?.container;
-  const headerClassName =
-    typeof className === 'string' ? '' : className?.header;
+  const { left = 'Cancel', right = 'Submit' } = copy;
+  const titleElement = (
+    <Title
+      className={classNames(
+        !isTitleBesideCancel && 'mt-5',
+        className?.title ?? 'typo-body',
+      )}
+    >
+      {title}
+    </Title>
+  );
 
   return (
-    <div className={classNames('flex flex-col', containerClassName)}>
+    <div className={classNames('flex flex-col', className?.container)}>
       <div
         className={classNames(
-          'flex flex-row justify-between border-b border-border-subtlest-tertiary px-4 py-2',
-          headerClassName,
+          'flex flex-row items-center border-b border-border-subtlest-tertiary px-4 py-2',
+          className?.header,
         )}
         ref={headerRef}
       >
         <Button {...leftButtonProps} variant={ButtonVariant.Tertiary}>
-          {copy?.left ?? 'Cancel'}
+          {left}
         </Button>
+        {isTitleBesideCancel && title && titleElement}
```

Comment:
Can u attach an image of these 2 variations of title, just not picturing them right in my head

Reply:
The other one is the existing layout we have. For example:

![Screenshot 2024-04-25 at 6 20 14 PM](https://github.com/dailydotdev/apps/assets/13744167/a9deda4c-7c4f-48de-87a9-f541cefb94c1)

Reply:
Cool thank u, commented above about what I ment https://github.com/dailydotdev/apps/pull/3043#discussion_r1579357147